backend/typescript/rest/campRoutes.ts

```diff
@@ -168,6 +170,67 @@ campRouter.get("/csv/:id", async (req, res) => {
   }
 });
 
+campRouter.post(
+  "/:campId/form/",
+  createFormQuestionsValidator,
+  async (req, res) => {
+    try {
+      const successfulFormQuestions = await campService.createFormQuestions(
```

Comment:
hm I'm wondering if here we can use just use appendFormQuestions as well instead of createFormQuestions? the code for the two methods is almost the same except for one linee

Reply:
the difference is one purges the entire set of form questions for a fresh batch, while the other appends to an already existing list, we could just remove the create form questions endpoint and do everything off append, but im not sure if thats what is wanted...

Reply:
I think we'll only end up using the append, but let's just keep both for now!